notebooks/machine_learning/raw/tut8.ipynb

```diff
@@ -0,0 +1,479 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Classification\n",
+    "\n",
+    "Up to this point you've been using data to make real-valued predictions such as home prices. This type of modeling is called **regression**, hence the \"Regressor\" part of `RandomForestRegressor`. In general, regression means you are trying to predict real numbers like prices, failure rates, energy consumption, etc.\n",
```

Comment:
This "real" stuff is more academic than we need.

Let's just say `So far you have predicted targets that are numbers.`

You've already said regression is predicting a number, so if you want to keep the last sentence, I'd rephrase it to `In general, regression means you are trying to predict real numbers like ...` to "Other examples of regression include ..."

Because we just said regression means predicting numbers.